.github/workflows/ci.yml

```diff
@@ -58,3 +58,44 @@ jobs:
       run: |
         yum -y install gcc make
         make REDIS_CFLAGS='-Werror'
+
+  test-ubuntu-jemalloc:
+    runs-on: ubuntu-latest
+    if: github.repository == 'redis/redis'
+    timeout-minutes: 14400
+    steps:
+    - uses: actions/checkout@v2
+    - name: make
+      run: make REDIS_CFLAGS='-Werror -DREDIS_TEST'
+    - name: unittest
+      run: ./src/redis-server test all
+
+  test-ubuntu-32bit:
+    runs-on: ubuntu-latest
+    if: github.repository == 'redis/redis'
+    timeout-minutes: 14400
+    steps:
+    - uses: actions/checkout@v2
+    - name: make
+      run: |
+        sudo apt-get update && sudo apt-get install libc6-dev-i386
+        make 32bit REDIS_CFLAGS='-Werror -DREDIS_TEST'
+    - name: unittest
+      run: ./src/redis-server test all
+
+  test-valgrind:
+    runs-on: ubuntu-latest
+    if: github.repository == 'redis/redis'
+    timeout-minutes: 14400
+    steps:
+    - uses: actions/checkout@v2
+    - name: make
+      run: make valgrind REDIS_CFLAGS='-Werror -DREDIS_TEST'
+    - name: test
+      run: |
+        sudo apt-get update
+        sudo apt-get install tcl8.6 valgrind -y
+    - name: unittest
+      run: |
+        valgrind --track-origins=yes --suppressions=./src/valgrind.sup --show-reachable=no --show-possibly-lost=no --leak-check=full ./src/redis-server test all 2>> err.txt
+        if grep -q 0x err.txt; then cat err.txt; exit 1; fi
```

Comment:
why would cat print it twice?

p.s. maybe we wanna use `--log-file=` option for valgrind, instead of `2>` . not sure if the tests print anything meaningful to stderror